Src/TransmitData.c

```diff
@@ -7,18 +7,104 @@
 #include "FlightPhase.h"
 #include "Data.h"
 
-static int TRANSMIT_DATA_PERIOD = 250;
-
-void transmitData(
-    int32_t altitude,
-    int32_t epochTimeMsec,
-    int32_t latitude,
-    int32_t longitude,
-    int32_t pressure,
-    FlightPhase phase)
+static const int TRANSMIT_DATA_PERIOD = 250;
+
+static const uint8_t IMU_HEADER_BYTE = 0x31;
+static const uint8_t BAROMETER_HEADER_BYTE = 0x32;
+static const uint8_t GPS_HEADER_BYTE = 0x33;
+static const uint8_t OXIDIZER_TANK_HEADER_BYTE = 0x34;
+static const uint8_t COMBUSTION_CHAMBER_HEADER_BYTE = 0x35;
+static const uint8_t FLIGHT_PHASE_HEADER_BYTE = 0x36;
+
```

Comment:
What about Load Cell Data? Does this method only take care of Avionics data?

Reply:
The load cell data is from launch systems. There's some intra system communications within launch systems, avionics doesn't need to worry about it.

Reply:
Can we add the altitude as calculated by Nathan in the parachute control branch, and the elevation reported by the gps?  This will allow us to compare the accuracy of the altitude calculation. Log both of these as well on the sd card. 

Another important metric is the number of gps satellites we have fixed. That comes from the NMEA string. @rzayevan